std_msgs/Header header 
xsens_mti_driver/SatInfo[] satinfo
